fastapi_sqlalchemy/db.py

```diff
@@ -0,0 +1,21 @@
+"""Database engines management."""
+import sqlalchemy
+
+__ENGINE_REGISTRY = {}
```

Comment:
Lets make this an EngineRegistry a singleton class and then these functions can be class methods.

Reply:
Since this is going to be a class, it'll be easy to add a lock for synchronization.

Reply:
Will we benefit from a class? A module is a singleton by itself (loaded during import only once)